common/src/main/java/com/microsoft/identity/common/internal/broker/AuthUxJavaScriptInterface.kt

```diff
@@ -127,6 +130,9 @@ class AuthUxJavaScriptInterface {
             }
 
             val operation = parameters.operation
+            if (operation != null) {
```

Comment:
In what cases would this be null?

Reply:
I will let @fadidurah answer since it is a part of his changes.